scripts/run-postInstall.js

```diff
@@ -3,8 +3,8 @@ const { execSync } = require('child_process')
 const executionEnv = process.env.CI ? 'ci' : 'local'
 
 const postInstallCommands = {
-  local: 'patch-package && yarn-deduplicate --strategy=highest && yarn clean && gulp postinstall && yarn workspace @packages/server rebuild-better-sqlite3 && yarn build && yarn build-v8-snapshot-dev',
-  ci: 'patch-package && yarn clean && gulp postinstall && yarn workspace @packages/server rebuild-better-sqlite3',
+  local: 'patch-package && yarn-deduplicate --strategy=highest && yarn workspace @packages/server rebuild-better-sqlite3 && yarn build && yarn build-v8-snapshot-dev',
+  ci: 'patch-package && yarn workspace @packages/server rebuild-better-sqlite3',
```

Comment:
Ideally this would be a `lerna run` command with the scope. At some point we will want to get away from `yarn workspace`

Reply:
Updated.